pennylane/_device.py

```diff
@@ -421,7 +421,7 @@ def execute(self, queue, observables, parameters=None, **kwargs):
 
         Args:
             queue (Iterable[~.operation.Operation]): operations to execute on the device
-            observables (Iterable[~.operation.Observable]): observables to measure and return
+            observables (Iterable[~.measurements.MeasurementProcess]): observables to measure and return
```

Comment:
Can we avoid the breaking change in this release? E.g., support both `MeasurementProcesses` and `Observables`?

Reply:
This added complication will eventually be removed when the deprecation becomes a removal.

Reply:
Motivation: I think most of our plugins use `QubitDevice`, but I'd rather we avoid breaking `Device` for the sake of a deprecation.

Reply:
I'll give it a try. I think it's easier to just switch the PQ devices (I'm quite certain nothing else inherits from `Device` directly), and I don't feel so morally dirty doing it because the convention was not respected in the first place. Buuuut I'll see what I can do.

Reply:
I changed it to be consistent with my change to `QubitDevice.sample`, and now it's fine with observables or measurements.